### PR TITLE
add warpadjust

### PR DESCRIPTION
**Pull request by nadkarni-na**

### CODE REVIEW

**Comment by nadkarni-na** [comment]
@salma1601 can you review this (it's a very simple modification), we would like to use it in the next 2 days

File: sammba/registration/struct.py [hunk 1]
```diff
@@ -121,6 +121,11 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         raise ValueError(
             'Registration kind must be one of {0}, you entered {1}'.format(
                 registration_kinds, registration_kind))
+                
+    if registration_kind is 'nonlinear' and len(anat_filenames) < 5:
+        raise ValueError('At least 5 input files are required to make a ' +
```
Comment (salma1601) [inline sammba/registration/struct.py]: you don't need the `+` here

File: sammba/registration/struct.py [hunk 2]
```diff
@@ -121,6 +121,11 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         raise ValueError(
             'Registration kind must be one of {0}, you entered {1}'.format(
                 registration_kinds, registration_kind))
+                
+    if registration_kind is 'nonlinear' and len(anat_filenames) < 5:
+        raise ValueError('At least 5 input files are required to make a ' + 
+                         'template by non-linear \n registration. Only ' + 
+                         str(len(anat_filenames)) + ' have been provided.')
```
Comment (salma1601) [inline sammba/registration/struct.py]: It's better to use `'{0} have been provided.'.format(len(anat_filenames))`

File: sammba/registration/struct.py [hunk 3]
```diff
@@ -617,6 +625,11 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         out_tstat_warp_head = tstat(in_file=out_tcat.outputs.out_file,
                                     outputtype='NIFTI_GZ')
 
+        nwarp_adjusted_mean = 'warped_{0}_adjusted_mean.nii.gz'.format(n_iter)
```
Comment (salma1601) [inline sammba/registration/struct.py]: have you checked if this file is created in the `write_dir` ?

File: sammba/registration/struct.py (hunk 1 above)
Comment (nadkarni-na) [reply sammba/registration/struct.py]: OK will change

File: sammba/registration/struct.py (hunk 2 above)
Comment (nadkarni-na) [reply sammba/registration/struct.py]: OK will change

File: sammba/registration/struct.py (hunk 3 above)
Comment (nadkarni-na) [reply sammba/registration/struct.py]: yep it is

**Comment by salma1601** [comment]
you will need to update your branch, merging is blocked

**Comment by nadkarni-na** [comment]
just attempted it, will commit changes and deal with any necessary conflict resolutions tonight

**Comment by salma1601** [comment]
OK

**Pull request merged**